app/main.ipynb

```diff
@@ -100,18 +102,33 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "import numpy as np\n",
+    "\n",
+    "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
```

Comment:
The array includes a number (17.6) that is outside the specified range of 0 to 10.0. Please ensure all numbers are within this range as per the task requirement.